src/xrpl-api.js

```diff
@@ -21,29 +21,35 @@ const LEDGER_CLOSE_TIME = 1000
 
 class XrplApi {
 
-    #rippledServer;
+    #primaryRippledServer;
+    #fallbackRippledServers;
     #client;
     #events = new EventEmitter();
     #addressSubscriptions = [];
     #initialConnectCalled = false;
     #isPermanentlyDisconnected = false;
+    #isPrimaryServerConnected = false;
+    #isFallbackServerConnected = false;
+    #xrplClientOptions;
     #autoReconnect;
 
     constructor(rippledServer = null, options = {}) {
-        this.#rippledServer = rippledServer || DefaultValues.rippledServer;
-        this.#initXrplClient(options.xrplClientOptions);
+        this.#primaryRippledServer = rippledServer || DefaultValues.rippledServer;
+        this.#fallbackRippledServers = options.fallbackRippledServers || []; //Default fallback server list should be defined here.
```

Comment:
Let's add [] in `DefaultValues.fallbackRippledServers` in `DefaultValues` objects and use it here. So later we can add list later.